apps/expo/tailwind.config.ts

```diff
@@ -5,4 +5,11 @@ import baseConfig from "@cfd/tailwind-config";
 export default {
   content: ["./src/**/*.{ts,tsx}"],
   presets: [baseConfig],
+  theme: {
+    extend: {
+      fontFamily: {
+        poppins: ["Poppins_400Regular", "sans-serif"],
```

Comment:
since we're only using poppins 400 and poppins 500, maybe the best way to get around the font weight issue is just to have a font-poppins500 and a font-poppins400?